cfme/common/__init__.py

```diff
@@ -1,13 +1,18 @@
 # -*- coding: utf-8 -*-
+from collections import namedtuple
 from functools import partial
 from urlparse import urlparse
 
 from cached_property import cached_property
 from cfme.fixtures import pytest_selenium as sel
 from cfme.web_ui import CheckboxTree, flash, form_buttons, mixins, toolbar
+from sqlalchemy.orm import aliased
 from utils import attributize_string, version
+from utils.db import cfmedb
+from utils.varmeth import variable
 
 pol_btn = partial(toolbar.select, "Policy")
+Tag = namedtuple('Tag', 'category, tag_name single_value')
```

Comment:
@mfalesni I see `Tag` class from `Configuration` has action methods. I could not get how it is working and how `single_value` managed here. Could you please point me some document for this `Tag` class?
https://github.com/ManageIQ/integration_tests/blob/master/cfme/configure/configuration/__init__.py#L1770